otodom/task_1/Valentyn/src/Scraper.py

```diff
@@ -0,0 +1,82 @@
+import requests
+from bs4 import BeautifulSoup, Tag
+from HouseItem import HouseItem
+
+
+class Scraper:
+    TARGET_URL = "https://www.otodom.pl/pl/wyniki/sprzedaz/mieszkanie/mazowieckie/warszawa/warszawa?limit=72"
+    headers = {
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3"
+    }
+
+    def __init__(self):
+        self.itemLinksVisited = []
```

Comment:
https://cs50.harvard.edu/python/2022/psets/2/camel/

in python it is recommended to use snake case for variable, function etc, names.

Reply:
Resolved